hcp/handler.go

```diff
@@ -0,0 +1,33 @@
+package hcp
+
+import (
+	"context"
+
+	sdkpacker "github.com/hashicorp/packer-plugin-sdk/packer"
+	"github.com/hashicorp/packer/hcl2template"
```

Comment:
This lint error appears to be an issue with golangci-lint v1.42.0 < 1.48.0, which is bit using Go 1.17 and not Go1.18. 
I am able to reproduce locally. Updating to 1.48.0 does not report an issue. I suspect that it may be do to some of the Go dependencies.

Reply:
True, the PR I uploaded before that bumps the golint-ci should take care of this one here.